products/libllbuild/BuildSystem-C-API.cpp

```diff
@@ -351,7 +351,7 @@ class CAPIBuildSystemFrontendDelegate : public BuildSystemFrontendDelegate {
       }
     }
 
-    llb_build_key_t** data() { return (llb_build_key_t **)&keys[0]; }
+    llb_build_key_t** data() { return (llb_build_key_t **)keys.data(); }
```

Comment:
Does this actually fix the problem?

From `[1]`:
> If [size()](https://en.cppreference.com/w/cpp/container/vector/size) is ​0​, data() may or may not return a null pointer.

`[1]`: https://en.cppreference.com/w/cpp/container/vector/data